_posts/2024-01-12-call-for-proposals-for-djangocon-us-2024-website.md

```diff
@@ -54,12 +66,12 @@ DjangoCon US will need:
 * City and date information
 * Sponsorship submission form hosted through third-party
 * Landing page copy
-* The 2023 website which shows a complete and working implementation of what we are looking for
+* The [2023 website](https://github.com/djangocon/2023.djangocon.us/) which shows a complete and working implementation of what we are looking for
 * Technical assistance
 
 ## Full Website Deliverable
 
-* Delivery of the full website design
+* Delivery of the full website design and ideally a starter code base including layouts for the pages listed in the deliverables section
```

Comment:
I'm struggling with this wording still. We are asking for the full website design (side note: we should spell out this is an HTML + associated files), but then I think "ideally" makes the rest of this come off as optional. 

For a proposal, optional means we shouldn't expect to get that, and I guess my question is what we really want? cc @sodevious

Reply:
I'm open to either-or, depending on the proposal. I'd like to hear what they can do

Reply:
I'd like to maybe put it as a "strongly preferred" but leaving it open to still being an option if someone comes up with a sublime design that we can implement ourselves without too much work